versioned_docs/version-2.0/tutorials/rag.mdx

```diff
@@ -0,0 +1,545 @@
+```python
```

Comment:
Also need to rename this file to "RAG.mdx" (capitalized) - otherwise it shows up incorrectly as all lower-cased in the sidebar